builtin/logical/aws/secret_access_keys.go

```diff
@@ -69,11 +68,10 @@ func genUsername(displayName, policyName, userType, usernameTemplate string) (re
 		// To prevent template from exceeding IAM length limits
 		if len(ret) > 64 {
 			ret = ret[0:64]
-			warning = "the calling token display name/IAM policy name were truncated to 64 characters to fit within IAM username length limits"
+			warning = fmt.Sprintf("the calling token's %s user name was truncated to 64 characters to fit within username length limits", userType)
 		}
 	case "sts":
-		// Capped at 32 chars, which leaves only a couple of characters to play
-		// with, so don't insert display name or policy name at all
+		// Capped at 32 chars
```

Comment:
Can we move this comment to be right above where we declare `defaultSTSTemplate` since it's no longer obvious here??

Reply:
Updated